src/components/PermissionsModal/PermissionsModal.js

```diff
@@ -0,0 +1,164 @@
+import React, { useMemo, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Pane, Modal, Paneset, PaneHeader, Loading, Layout } from '@folio/stripes/components';
```

Comment:
```suggestion
import React, {
  useMemo,
  useState,
} from 'react';
import PropTypes from 'prop-types';
import {
  Pane,
  Modal,
  Paneset,
  PaneHeader,
  Loading,
  Layout,
} from '@folio/stripes/components';
```

Reply:
And also look through all the imports in the others files

Reply:
@OleksandrHladchenko1 thanjs for CHANGELOG catching. Imports were updated as well